google-drive.js

```diff
@@ -172,7 +169,7 @@ function downloadFile(resolve, reject) {
       })
 
     }).catch(e => {
-      console.log('ERRORED', e)
+      addToLog('ERRORED', e)
```

Comment:
Please only convert `index.js` file; `addToLog` method is not present in this file